phpstan-console-application.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
This file is for tests? If yes, you should move it to the `tests` directory

Reply:
This is the same as admin-bundle

Reply:
Yes, since it's phpstan-related, it's better to keep it with phpstan files.

Reply:
Okay, please exclude this file from the archive using `.gitattributes`

Reply:
https://github.com/sonata-project/dev-kit/pull/1588

Reply:
Merged and fixed.